HumanNeverDie/Projects/Features/HistoryFeature/Sources/Calendar/View/AMDCalendarCommonViews.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  AMDCalendarCommonViews.swift
+//  HistoryFeature
+//
+//  Created by Seulki Lee on 6/27/25.
+//
+
+import SwiftUI
+import DesignSystem
+
+public struct CalendarTitleView: View {
+  let title: String
+  let onTap: () -> Void
+
+  public init(title: String, onTap: @escaping () -> Void) {
+    self.title = title
+    self.onTap = onTap
+  }
+
+  public var body: some View {
+    HStack {
+      Button(action: onTap) {
+        HStack(spacing: 4) {
+          Text(title)
+            .amdFont(.xlargeBold)
+            .foregroundColor(Color.gray100)
+
+          Image(systemName: "chevron.down")
+            .foregroundColor(Color.gray50)
```

Comment:
현재 `ShapeStyle`에 대한 **constrained extension**으로 디자인 시스템의 컬러가 선언되어 있어, `Color` 타입 추론이 되고 있는 상태에요!

아래처럼 `Color.` 접두어 없이 `.gray50`으로 사용하는게 코드가 더 간결해지고 가독성도 향상될 것 같습니다.
```suggestion
            .foregroundColor(.gray50)
```

다른 코드에서도 `Color.` 접두어가 필요한 지 한번 점검해보세요~!

Reply:
네 맞습니다!  constrained extension으로 접두어 없이도 되죠... 
말씀해주신 대로 가독성과 코드 간결성 측면에서 훨씬 나은 것 같아요!
해당 부분 수정했고, 다른 코드들도 `Color.` 접두어가 불필요하게 붙은 부분이 있는지 전체적으로 점검 후 적용하였습니다.!
감사합니다!!!